### PR TITLE
Add sqlflow playground api server

### PR DESCRIPTION
**Pull request by lhw362950217**

### CODE REVIEW

File: python/README.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+# SQLFlow Playground Server
```
Comment (Yancey0623) [inline python/README.md]: It's an excellent idea, maybe we should push a design doc to introduce the Playground server and what APIs the server exposed, and then push  the implementation PRs.

please @typhoonzero also takes a  look at  the PR.

Comment (typhoonzero) [reply python/README.md]: Why not use sqlflowserver grpc service directly?

Comment (typhoonzero) [reply python/README.md]: Is this used for aliyun DSW only?

Comment (lhw362950217) [reply python/README.md]: This service is used to manage playground resource. As for those which can directly connect to sqlflowserver, this service is not necessary.
Current it's only used by DSW. We need to create a DB pod for each user, and kill the pod after some idle time. Of course, this use pattern is not only for DSW.

Comment (lhw362950217) [reply python/README.md]: done

**Review by Yancey0623 — APPROVED** [review]
LGMT!

**Pull request merged**